src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  box-sizing: border-box;
+  margin-top: 0;
+}
+
+.search {
+  margin: 20px 0;
+}
+
+.search-input {
+  width: 100%;
+  box-sizing: border-box;
+  height: 70px;
+  padding: 25px 20px 23px 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  outline: none;
+}
+
+.bigSearch-input {
+  background: url("images/Search.svg");
+  background-position: 25px 23px;
+  background-repeat: no-repeat;
+  background-size: 19px;
+}
+
+.search-input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
```

Comment:
Placeholder will inherit these styles from parent element, move this styles to main class
```suggestion
```